
<p>
<link rel="stylesheet" type="text/css" href="resume.css">
</p>

<h1 id="coleman-broaddus">Coleman Broaddus</h1>

<table>
	<tbody>
		<tr class="odd">
			<td align="left">Regerstraße 2</td>
			<td align="left">coleman.broaddus@gmail.com</td>
		</tr>
		<tr class="even">
			<td align="left">Dresden, DE 01309</td>
			<td align="left"></td>
		</tr>
	</tbody>
</table>


<h1 id="education">Education</h1>
<dl>
	<dt>2005-2010</dt>
	<dd><strong>BSc, Physics</strong>; Carnegie Mellon University (Pittsburgh, PA)
	</dd>
	<dt>2011-2015</dt>
<dd>
<p>
	<strong>MSc, Nanobiophysics</strong>; Dresden University of Technology (Dresden, DE)
</p>

<p>
	<em>Thesis title: Pattern formation in chemically reactive, phase-separating systems</em>
</p>
	</dd>
	<dt>2015-present</dt>
<dd>
<p>
	<strong>Phd, Computer Science</strong>; Max Planck Institute for Molecular Cell Biology and Genetics (Dresden, DE)
</p>

<p>
	<em>Thesis title: Tracking cells in images during development</em>
</p>
	</dd>
</dl>

<h1 id="research-experience">Research Experience</h1>

<p>
October 2010 - March 2011<br/>
<em>Brownian dynamics simulations of centrosome growth</em><br/>
Supervisor: David Zwicker; Lab: Jülicher Lab, MPI-PKS<br />
I wrote a Brownian Dynamics simulation of the centrosome growth process based on the model in [Zwicker et al. PNAS 2014]. 
This model included competition between centrosomes within a cell, arbitrary cell shape, and accurate capturing of stochaticity and finite number effects.
</p>

<p>
March 2013 - October 2013 (Masters Thesis towards M.Sc. Nanobiophysics)<br />
<em>Pattern Formation in chemically reactive, phase-separating systems</em><br />
Supervisor: Frank Jülicher; Lab: Jülicher Lab, MPI-PKS<br />
I wrote my masters thesis on a simpler yet larger class of systems composed of binary mixtures with two chemical species that phase-separate and undergo (first-order and/or autocatalytic) chemical reactions. 
We ran simulations on a lattice, observing Lifshitz-Slyozov-like scaling behavior, but with interesting finite-lengthscale steady state solutions which result directly from the interplay of the reactions with the phase separation.
</p>

<p>
July 2014 - December 2014<br />
<em>A model of colony formation based on Kinetic Theory</em><br />
Supervisor: Christoph Weber; Lab: Zaburdaev Lab, MPI-PKS<br />
There a many examples in ecology of organism that form <em>colonies</em> as a response to environmental pressure. 
Christoph developed a dynamic density field model for colony formation that includes diffusion and nonlocal binary interactions, resembling the Boltzmann equation from Kinetic Theory. 
I ran particle-based simulations of this model in order to observe the dynamics, and to determine the long-time stability and symmetry properties of the colony phase.
</p>

<p>
October 2014, March 2015 - June 2015<br />
<em>Novel cell segmentation method applied to fluorescent images of zebrafish</em><br />
Supervisors: Dagmar Kainmueller and Florian Jug; Lab: Myers Lab, MPI-CBG<br />
We developed a technique for segmenting fluorescence microscopy images of early developmental (e.g. blastula) tissues. 
The technique was applied to cryoslices of early zebrafish embryos with between 50 and 200 cells per slice, spanning a wide range of 2d cell geometries. 
This segmentation procedure works as a Fiji plugin, and will be made available to the Fiji community. 
This is a part of a larger work towards semi-automated mapping of single molecule, cellular-level gene expression across whole tissues in developing zebrafish [Stapel et al., submitted to Development on 20-7-2015].
</p>

<p>
November 2015 - Present (Doctoral work towards PhD in Computer Science)<br />
<em>Tracking cells during development</em><br />
Supervisor: Gene Myers<br />
Understanding when and where individual cells move, divide and die during embryogenesis is a fundamental part of modern developmental biology. 
Current fluorescence microscopy allows us in principle to track the movement of cells throughout the entire embryo and across the full timespan of development for a variety of model species. 
However, so far automated cell tracking algorithms still struggle to deal with tracking during development as it poses many qualitatively new challenges to the field of cell tracking. 
During my PhD I have focused on new models and algorithms for cell tracking during development and the upstream tasks of cell detection and segmentation.
</p>

<h1 id="software">Software</h1>
<ul>
	<li>Languages: Have been payed to write code in python, c, clojure, and java.</li>
	<li>Machine learning frameworks: ilastik, weka, scikit-learn, keras and tensorflow</li>
</ul>

<h1 id="publications">Publications</h1>

<p>Broaddus C, Weigert M, Schmidt U, Myers G. <b>Cell nuclei detection and segmentation from weak annotations.</b> ISBI 2019. (Submitted).</p>
<p>Schmidt U, Weigert M, Broaddus C, Myers G. <b>Cell Detection with Star-convex Polygons.</b> MICCAI 2018.</p>
<p>Weigert, Martin, et al. <b>Content-aware image restoration: pushing the limits of fluorescence microscopy.</b> Nature Methods 15 (2018). 236463.</p>
<p>Stapel LC, Broaddus C, Vastenhouw NL. <b>Detection and Automated Analysis of Single Transcripts at Subcellular Resolution in Zebrafish Embryos.</b> InRNA Detection 2018 (pp. 143-162). Humana Press, New York, NY.</p>
<p>Stapel LC, Lombardot B, Broaddus C, Kainmueller D, Jug F, Myers EW, Vastenhouw NL. <b>Automated detection and quantification of single RNAs at cellular resolution in zebrafish embryos.</b> Development. 2016 Feb 1;143(3):540-6.</p>
